dialog.html

```diff
@@ -48,6 +70,13 @@ <h2></h2>
         <div id="video-placeholder" class="video"></div>
     </div>
 
+
+    <!-- scripts -->
+    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="dialog.js" type="text/javascript"></script>
+    <script src="https://apis.google.com/js/client.js?onload=init"></script>
```

Comment:
Do we need to make local copies of these scripts?

Reply:
This is just the search/play YouTube URL dialog. I can change that. I'm not exactly sure if we need local copies of the scripts.

Reply:
Yes, download all scripts and reference them locally. Some of these may already be in place.

Reply:
I'm not quite sure how to do that. Do I download the scripts and add those files to this branch? Do I need to change anything about the script tags?